setup.py

```diff
@@ -71,9 +71,12 @@ def get_long_description():
         'xlrd>=1.0.0',
     ],
     extras_require={
+        'sbert': ['sentence_transformers'],
+        'doc2vec': ['gensim'],
         'dev': ['check-manifest'],
         'test': ['coverage'],
         'performance': ['python-Levenshtein'],
+        'all': ['gensim', 'python-Levenshtein', 'sentence_transformers'],
```

Comment:
Can you make this into something more robust? 'all' is not incomplete, isn't it?

Maybe define some global:

```
DEPS_TEST = ['pytest', 'coverage']
DEPS_DEV = ['check-manifest']

extras_require={
   ...,
   'all': DEPS_TEST + DEPS_DEV + ....
```